ynr/apps/moderation_queue/tests/test_upload_photo.py

```diff
@@ -93,36 +116,36 @@ def test_shows_photo_policy_text_in_photo_upload_page(self):
         response = self.app.get(upload_form_url, user=self.test_upload_user)
         self.assertContains(response, "Photo policy")
 
-    def test_only_one_photo_in_queue_per_person(self):
-        # create a queued image for person 2009
-        QueuedImage.objects.create(
-            person_id=2009,
-            user=self.test_upload_user,
-            why_allowed="copyright-assigned",
-            justification_for_use="I took this photo",
-        )
-        # assert that there is alread one photo in the queue for this person
-        queued_images = QueuedImage.objects.filter(person__id=2009)
-        self.assertEqual(queued_images.count(), 1)
-        # try to upload another image to the queue for person 2009
-        upload_form_url = reverse("photo-upload", kwargs={"person_id": "2009"})
-        form_page_response = self.app.get(
-            upload_form_url, user=self.test_upload_user
-        )
-        form = form_page_response.forms["person-upload-photo-image"]
-        with open(self.example_image_filename, "rb") as f:
-            form["image"] = Upload("pilot.jpg", f.read())
-        form["why_allowed"] = "copyright-assigned"
-        form["justification_for_use"] = "I took this photo"
-        upload_response = form.submit()
-        # assert validation error is shown
-        self.assertContains(
-            upload_response,
-            "An image for this person has already been uploaded to the moderation queue.",
-        )
-        # assert that there is still only one photo in the queue for this person
-        queued_images = QueuedImage.objects.filter(person__id=2009)
-        self.assertEqual(queued_images.count(), 1)
+    # def test_only_one_photo_in_queue_per_person(self):
```

Comment:
Why is this commented out, is there a problem with the test?

Reply:
This test relates to the validation which will be removed

Reply:
Based on what we discussed about leaving existing queue images as they are, I don't think we want to test that each person just has one image in the queue?

Reply:
Yep makes sense!